### PR TITLE
Pupulate MTAirCompressor

### PR DESCRIPTION
**Pull request by pkubanek**
- Added CSC
- Timer info
- Read status
- improved error handling
- command line arguments

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -2,6 +2,7 @@
 ts_AirCompressors
 #################
 
+Communication with pneumatic air compressors. Collect compressors telemetry.
```
Comment (wvreeven) [inline README.rst]: Please use one sentence per line.

File: bin/run_mtaircompressor.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+# This file is part of AirCompressors.
```
Comment (wvreeven) [inline bin/run_mtaircompressor.py]: Please change `AirCompressors` to the project name `ts_MTAirCompressor` here and in all other files.

File: python/lsst/ts/MTAirCompressor/aircompressor_csc.py [hunk 3]
```diff
@@ -0,0 +1,262 @@
+# This file is part of ts_AirCompressors.
```
Comment (wvreeven) [inline python/lsst/ts/MTAirCompressor/aircompressor_csc.py]: Please change `ts_AirCompressors` to the project name `ts_MTAirCompressor` here and in all other files.

File: python/lsst/ts/MTAirCompressor/aircompressor_csc.py [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+#
+# Developed for the LSST Data Management System.
```
Comment (wvreeven) [inline python/lsst/ts/MTAirCompressor/aircompressor_csc.py]: Please change `LSST Data Management System` to `Vera Rubin Observatory Telescope and Site Systems` here and in all other files.

File: python/lsst/ts/MTAirCompressor/aircompressor_csc.py [hunk 5]
```diff
@@ -0,0 +1,262 @@
...
+        if modbus_exception.original_code == 4:
+            message = (
+                f"Cannot address 0x{address:04x}: {modbus_exception.exception_code}"
+            )
+        elif modbus_exception.original_code == 6:
+            message = f"Cannot write register address 0x{address:04x}: {modbus_exception.exception_code}"
+        else:
+            message = (
+                f"Cannot call function {modbus_exception.function_code} : "
+                "{modbus_exception.exception_code}, address 0x{address:04x}"
```
Comment (wvreeven) [inline python/lsst/ts/MTAirCompressor/aircompressor_csc.py]: There is an `f` missing right before the first `"`.

File: python/lsst/ts/MTAirCompressor/aircompressor_csc.py [hunk 6]
```diff
@@ -0,0 +1,262 @@
...
+        self.hostname = (
+            f"m1m3cam-aircomp{kwargs['index']:02d}.cp.lsst.org"
+            if args.hostname is None
+            else args.hostname
+        )
+        self.unit = kwargs["index"] if args.unit is None else args.unit
+
+    async def do_start(self, data):
+        await super().do_start(data)
+        # loop, self.client = ModbusClient(
```
Comment (wvreeven) [inline python/lsst/ts/MTAirCompressor/aircompressor_csc.py]: Is there a reason to keep the commented out code here? If yes, please add a comment why. If no, please remove.

**Review by wvreeven — CHANGES_REQUESTED** [review]
In general this looks good but I have added several comments so please review those. Also please add docstrings to all methods in `aircompressor_csc.py`. Finally, I really don't see the point of asking 4 people to review your PR. Please just pick one in the future.

File: bin/run_mtaircompressor.py (hunk 2 above)
Comment (pkubanek) [reply bin/run_mtaircompressor.py]: Changed (hopefully everywhere). Was caused by originally naming it AirCompressors

File: python/lsst/ts/MTAirCompressor/aircompressor_csc.py (hunk 6 above)
Comment (pkubanek) [reply python/lsst/ts/MTAirCompressor/aircompressor_csc.py]: Removed, note added why async doesn't make sense (now),

File: bin/run_mtaircompressor.py [hunk 7]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+# This file is part of ts_MTAirCompressors.
```
Comment (wvreeven) [inline bin/run_mtaircompressor.py]: The project is `ts_MTAirCompressor` so please correct this here.

**Review by wvreeven — APPROVED** [review]
Looks great now. Please review my only comment before merging. Thanks!

**Pull request merged**